### ISSUE TITLE
Multi-arch support

### ISSUE THREAD

**Issue by diederichm** [IBM/cloud-pak#329]
Ibm Operator Catalog Enablement v1.0.0
does not come up on openshift 4.4.9 on s390x and on ppc64le
installed via GUI (OCP+)

this is ppc64le
[root@lozpower-inf ~]# oc logs ibm-operator-catalog-dxr2q -n openshift-marketplace
standard_init_linux.go:211: exec user process caused "exec format error"

s390x is similar

**Comment by huizengaJoe** [comment]
@cdjohnson @nathanbrophy  Guys correct me if I am wrong, the catalog images are amd64 only right now.  Redhat claims to support multiarch catalog images in ocp 4.5 and Nathan you are testing that out right now, right?

**Comment by huizengaJoe** [comment]
https://github.com/operator-framework/operator-registry/issues/208

**Comment by eponafyrefly** [comment]
Yes, I have a cu reporting in 4.4.13 he was only able to see the elasticsearch operator on ppc64le.  He upgraded to 4.5 and sees a few more, but def not the 200+ available to other architectures.  This was tough to chase down as I don't have other archs available to test on in my lab.  What's odd is oc get packagemanifests showed him everything, so I went down a console rabbit hole for a bit.
